.github/CONTRIBUTING.md

```diff
@@ -133,7 +136,18 @@ logically independent patches which build on each other.  This makes large
 changes easier to review and approve which speeds up the merging process.
 * Try to keep pull requests simple. Simple code with comments is much easier
 to review and approve.
+* All proposed changes must be approved by a ZFS on Linux organization member.
+
+#### Tests and Benchmarks
+* Every pull request will by tested by the buildbot on multiple platforms by running the [zfs-tests.sh](
```

Comment:
If these go to the same place, it might be reasonable to link as `[zfs-tests.sh and zloop.sh](https://github.com/zfsonlinux/zfs/wiki/Building-ZFS#running-zloopsh-and-zfs-testssh)`